packages/gatsby-plugin-page-creator/package.json

```diff
@@ -21,12 +21,10 @@
   "dependencies": {
```

Comment:
Not 100% sure but I think the reason why e2e tests are failing are because it can't find the `gatsby-page-utils` package? Make sure to add it do the dependencies here.

It's just a hunch, thanks for taking this on!